### DISCUSSION TITLE
Protractor to WDIO migration

### DISCUSSION THREAD

**Discussion by AnilCa1905** [webdriverio/codemod#77] [Migration Issue]
Is there an easy way to migrate the scripts from protractor to wdio?
the config files can be migrated using codemod but how to migrate the protractor scripts to WDIO format?

**Comment by christian-bromann** [comment]
> the config files can be migrated using codemod but how to migrate the protractor scripts to WDIO format?

Same way, call
```sh
npx jscodeshift -t ./node_modules/@wdio/codemod/protractor ./e2e/
```

where `./e2e/` contains your protractor test files.
